metaflow/plugins/tagging_cli.py

```diff
@@ -0,0 +1,303 @@
+import click
+from metaflow import namespace
+from metaflow.util import resolve_identity
+from metaflow.exception import CommandException, MetaflowNotFound
+from metaflow.exception import MetaflowNamespaceMismatch
+
+# a rough emulation of itertools.zip_longest for our needs
```

Comment:
you could move this to `util` which centralizes py2/py3 compatibility hacks. It allows you to use `itertools.zip_longest` for Py3 natively. Add it here https://github.com/Netflix/metaflow/blob/master/metaflow/util.py#L16

Reply:
THis is actually no longer used I think so I will remove.